java-frontend/src/test/java/org/sonar/java/model/expression/InstanceOfTreeImplTest.java

```diff
@@ -78,23 +78,6 @@ void test_GuardedPatternInstanceOfTree() {
 
   }
 
-  @Test
```

Comment:
I'm ok with dropping this one, but what about keeping one with parentheses?
```java
ExpressionTree condition = ifCondition("o instanceof (String s) && s.length() > 10");
```

Reply:
Parenthesis around `String s`